src/shared/logger/activation.ts

```diff
@@ -43,6 +43,7 @@ export async function activate(
         {
             logPaths: [logUri.fsPath],
             outputChannels: [chan],
+            useConsoleLog: true,
```

Comment:
@nkomonen-amazon why does this default to true? does that change the nodejs behavior compared to before?

Reply:
Yeah this will have nodejs log to the vscode Developer Tools console as well

Reply:
some reasons to default to `useConsoleLog=false`

- it matches the previous behavior (for nodejs), less risk. (set to true when in browser-mode.)
- in cloud9 this will cause duplicate logs because cloud9 already forwards our logs to the browser console. 
- in the integ tests CI logs, it looks like this is causing duplicate messages:
  ```
  2023-09-07 13:42:23 [VERBOSE]: stderr: 
  Fetching public.ecr.aws/sam/build-java8.al2:latest-x86_64 Docker container image....
  2023-09-07 13:42:23 [VERBOSE]: stderr: 
  Fetching public.ecr.aws/sam/build-java8.al2:latest-x86_64 Docker container image....
  ```